gromacs/fileformats/blocks.py

```diff
@@ -0,0 +1,461 @@
+
+
+#
+# JD: adopted from 'pytopol' from https://github.com/resal81/PyTopol
+#
+
+class System(object):
+    def __init__(self):
+        self.molecules = tuple([])
+
+        self.atomtypes        = []
+        self.bondtypes        = []
+        self.nonbond_params   = []
+        self.angletypes       = []
+        self.dihedraltypes    = []
+        self.impropertypes    = []
+        self.cmaptypes        = []
+        self.interactiontypes = []
+        self.pairtypes        = []
+        self.constrainttypes  = []
+        self.forcefield=  None
+
+        self.information = {} # like 'atomtypes': self.atomtypes
+
+
+
+class Molecule(object):
+
+    def __init__(self):
+        self.chains    = []
+        self.atoms     = []
+        self.residues  = []
+
+        self.bonds     = []
+        self.angles    = []
+        self.dihedrals = []
+        self.impropers = []
+
+        self.cmaps     = []
+        self.pairs     = []
+        self.exclusion_numb = None  # 0, 1, 2, ..
+        self.virtual_sites3 = []
+        self.exclusions = []
+        self.settles    = []
+        self.constraints= []
+
```

Comment:
I'm confused: maybe it's not clear from the code but these are instance attributes not class attributes. I don't want my Molecule instances to share `atoms` and other class members with *other* Molecule objects. 

What you're proposing will cause
```
class Molecule:
    chains = []
    def __init__(self, *args, **kwargs): 
        pass

m1 = Molecule()
m2 = Molecule()
m1.chains.append("A")
print m2.chains
Out: ['A']
```

Maybe i'm getting something wrong but this doesn't look like the desired behaviour?